man/systemd.link.xml

```diff
@@ -364,6 +364,47 @@
           <xi:include href="version-info.xml" xpointer="v211"/>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Property=</varname></term>
+        <listitem>
+          <para>Set specified udev properties. This takes space separated list of key-value pairs
+          concatenated with equal sign (<literal>=</literal>). Example:
+          <programlisting>Property=HOGE=foo BAR=baz</programlisting>
+          This option can be specified multiple times. If an empty string is assigned, then the all previous
+          assignments are cleared.</para>
+
+          <xi:include href="version-info.xml" xpointer="v256"/>
```

Comment:
i think this should mention the `ID_NET_MANAGED_BY=` property explicitly, and that it makes sense to set this here.

something like this:

> This setting is useful to configure the `ID_NET_MANAGED_BY=` property which declares which network management service shall manage the interface, which is respected by `systemd-networkd` and others. Use `Property=ID_NET_MANAGED_BY=io.systemd.Network` to declare explicitly that `systemd-networkd` shall manage the interface, or set the property to something else to declare explicitly it shall <emphasis>not</emphasis> do so. See systemd.network(5) for details how this this property is used to match interface names.


you get the idea.